tools/nebula-spark/src/main/scala/com/vesoft/nebula/tools/connector/package.scala

```diff
@@ -6,27 +6,18 @@
 
 package com.vesoft.nebula.tools
 
-import org.apache.spark.graphx
 import org.apache.spark.graphx.{Edge, Graph, VertexId}
 import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.types.{DoubleType, IntegerType, StringType, StructField, StructType}
-import org.apache.spark.sql.{
-  DataFrame,
-  DataFrameReader,
-  DataFrameWriter,
-  Encoders,
-  Row,
-  SparkSession
-}
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter, Row, SparkSession}
 
 package object connector {
 
   type Address    = (String, Int)
   type EdgeRank   = Int
-  type Property   = (String, Any)
+  type Propertyy  = (String, Any)
```

Comment:
typo?

Reply:
The type Property is the same as nebula.data.Property, so changed it to Property temporary.

Reply:
why ？ the package name not the same

Reply:
Reader used nebula.data.Property in NebulaUtils.scala (NebulaUtils is in the same path with package), if i change type Propertyy to Property, it will use type Property automatically.